site/src/components/AuditLogRow/AuditLogDescription.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react"
+import { AuditLog } from "api/typesGenerated"
+import { Link as RouterLink } from "react-router-dom"
+import Link from "@material-ui/core/Link"
+import { makeStyles } from "@material-ui/core/styles"
+import i18next from "i18next"
+
+export const AuditLogDescription: FC<{ auditLog: AuditLog }> = ({
+  auditLog,
+}): JSX.Element => {
+  const classes = useStyles()
+  const { t } = i18next
+
+  let target = auditLog.resource_target.trim()
+
+  // audit logs with a resource_type of workspace build use workspace name as a target
+  if (
+    auditLog.resource_type === "workspace_build" &&
+    auditLog.additional_fields.workspaceName
+  ) {
+    target = auditLog.additional_fields.workspaceName.trim()
+  }
+
+  // SSH key entries have no links
+  if (auditLog.resource_type === "git_ssh_key") {
+    return (
+      <span>
+        {auditLog.description
+          .replace("{user}", `${auditLog.user?.username.trim()}`)
+          .replace("{target}", `${target}`)}
+      </span>
+    )
+  }
+
+  const truncatedDescription = auditLog.description
+    .replace("{user}", `${auditLog.user?.username.trim()}`)
+    .replace("{target}", "")
+
+  return (
+    <span>
+      {truncatedDescription}
+      {auditLog.resource_link ? (
+        <Link component={RouterLink} to={auditLog.resource_link}>
+          <strong>{target}</strong>
+        </Link>
```

Comment:
Having a link on workspace name for `workspace_build` strings is a little awkward - it might be better if this link was built around the word `build` instead of on the workspace target. However, that add a surprising amount of complexity, so I figured this was okay for now. If you object, let me know!